<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QMainWindow" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>588</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1042</width>
    <height>588</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1042</width>
    <height>588</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Raleway Black</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Registro de investigadores</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/Images/if_Turtle_204384.ico</normaloff>:/Images/if_Turtle_204384.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(206, 241, 158);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gpbUsuario">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>981</width>
      <height>550</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>981</width>
      <height>550</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>981</width>
      <height>550</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Raleway</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(241, 241, 241);</string>
    </property>
    <property name="title">
     <string>Registro de investigadores</string>
    </property>
    <widget class="QLineEdit" name="lneApm">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>300</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
</string>
     </property>
     <property name="maxLength">
      <number>30</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lneApp">
     <property name="geometry">
      <rect>
       <x>479</x>
       <y>300</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="maxLength">
      <number>30</number>
     </property>
    </widget>
    <widget class="QLabel" name="lblApp">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>300</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Apellido paterno</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lneNombre">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>300</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="maxLength">
      <number>30</number>
     </property>
    </widget>
    <widget class="QLabel" name="lblNombre">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>300</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Nombre</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblApm">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>300</y>
       <width>161</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Apellido materno</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnePassword_2">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>340</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="maxLength">
      <number>16</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lblPassword_2">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>340</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Confirmar contraseña</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lneUsuario">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>340</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="maxLength">
      <number>16</number>
     </property>
    </widget>
    <widget class="QLabel" name="lblUsuario">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Usuario</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnePassword">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>340</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="maxLength">
      <number>16</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lblPassword">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>340</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Contraseña</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblTipo">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>380</y>
       <width>181</width>
       <height>18</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Tipo de investigador</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dteNacimiento">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>380</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>1998</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="maximumDate">
      <date>
       <year>2018</year>
       <month>12</month>
       <day>31</day>
      </date>
     </property>
     <property name="minimumDate">
      <date>
       <year>1940</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
     <property name="displayFormat">
      <string>dd/MM/yyyy</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblCump">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>380</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Fecha nacimiento</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cboTipo">
     <property name="geometry">
      <rect>
       <x>559</x>
       <y>380</y>
       <width>391</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="psbAgregar">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>440</y>
       <width>170</width>
       <height>65</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>65</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton#psbAgregar {
background-color: rgb(120, 190, 151);
 }

 QPushButton#psbAgregar:hover {
background-color: rgb(56, 134, 136);
color:  rgb(255, 255, 255);
 }</string>
     </property>
     <property name="text">
      <string>Agregar</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/Images/user_add.ico</normaloff>:/Images/user_add.ico</iconset>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pbOut">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>440</y>
       <width>170</width>
       <height>65</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>65</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton#pbOut {
background-color: rgb(120, 190, 151);
 }

 QPushButton#pbOut:hover {
background-color: rgb(56, 134, 136);
color:  rgb(255, 255, 255);
 }</string>
     </property>
     <property name="text">
      <string>  Salir</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/Images/door_out.ico</normaloff>:/Images/door_out.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="lbImage">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>561</width>
       <height>191</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="images2.qrc">:/cuidador.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="lbDescription">
     <property name="geometry">
      <rect>
       <x>586</x>
       <y>70</y>
       <width>381</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Los investigadores son parte fundamental
en este proyecto, cuyo deber será llevar
un mayor control sobre el monitoreo
constante de las tortugas.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>lneApp</zorder>
    <zorder>lblApp</zorder>
    <zorder>lneNombre</zorder>
    <zorder>lblNombre</zorder>
    <zorder>lblApm</zorder>
    <zorder>lnePassword_2</zorder>
    <zorder>lblPassword_2</zorder>
    <zorder>lneUsuario</zorder>
    <zorder>lblUsuario</zorder>
    <zorder>lnePassword</zorder>
    <zorder>lblPassword</zorder>
    <zorder>lblTipo</zorder>
    <zorder>dteNacimiento</zorder>
    <zorder>lblCump</zorder>
    <zorder>cboTipo</zorder>
    <zorder>psbAgregar</zorder>
    <zorder>pbOut</zorder>
    <zorder>lbImage</zorder>
    <zorder>lbDescription</zorder>
    <zorder>lneApm</zorder>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lneNombre</tabstop>
  <tabstop>lneApp</tabstop>
  <tabstop>lneApm</tabstop>
  <tabstop>lneUsuario</tabstop>
  <tabstop>lnePassword</tabstop>
  <tabstop>lnePassword_2</tabstop>
  <tabstop>dteNacimiento</tabstop>
  <tabstop>cboTipo</tabstop>
 </tabstops>
 <resources>
  <include location="images2.qrc"/>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbOut</sender>
   <signal>clicked()</signal>
   <receiver>Register</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>714</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>684</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
